tests/unit/client.py

```diff
@@ -505,6 +505,21 @@ def test_port_detach_nic_error(self):
         with pytest.raises(FailedAPICallException):
             C.port.detach_nic('mock-01', 'gi1/1/8')
 
+    def test_show_port(self):
+        # do show port on a port that's not registered yet
+        with pytest.raises(FailedAPICallException):
+            C.port.show('mock-01', 'gi1/1/8')
+
+        C.port.register('mock-01', 'gi1/1/8')
+        assert C.port.show('mock-01', 'gi1/1/8') == {}
+        C.port.connect_nic('mock-01', 'gi1/1/8', 'node-09', 'eth0')
+        assert C.port.show('mock-01', 'gi1/1/8') == {
+                u'node': u'node-09', u'nic': u'eth0', u'networks': {}}
```

Comment:
Same thing here, I have a hard time scanning dictionaries where you have multiple key/value pairs on one line.